### PR TITLE
Remove unnecessary custom events

### LINKED ISSUE #71: Remove unnecessary events in ModalView
Currently the ModalView uses custom events to signal a button click. In this case the event handler in the controller can better be attached directly to the button to prevent unnecessary hassle and overhead.

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Removed unnecessary custom events and moved creation logic to a new controller
Also moved validation to controllers
fixes #71

### CODE REVIEW

**Comment by Markvanderuit** [comment]
Nothing to comment, looks good. Second member can merge.

File: src/main/java/control/CreationModalViewController.java [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+            });
+
+        addCameraShotCreationParsingHandlers();
+    }
+
+    /**
+     * Add handlers for parsing quarters to the camera shot creation view.
+     */
+    private void addCameraShotCreationParsingHandlers() {
+        // Add listeners for parsing to startfield
```
Comment (mennoo1996) [inline src/main/java/control/CreationModalViewController.java:45]: Don't you want to extract the handlers to seperate methods?

Comment (bart-de-jonge) [inline src/main/java/control/CreationModalViewController.java:45]: sure

File: src/main/java/control/ToolViewController.java [hunk 2]
```diff
@@ -34,106 +29,33 @@ public ToolViewController(ControllerManager controllerManager) {
...
      */
     private void initializeTools() {
-        new ToolButton("Add camerashot",
-                this.controllerManager.getRootPane().getRootHeaderArea(),
-                this::showCameraCreationWindow);
-        new ToolButton("Add directorshot",
-                this.controllerManager.getRootPane().getRootHeaderArea(),
-                this::showDirectorBlockCreationWindow);
+        // Toolbutton for creating a camerablock
+        ToolButton cameraBlockCreationTool = new ToolButton("Add camerashot",
```
Comment (mennoo1996) [inline src/main/java/control/ToolViewController.java]: Shouldn't you put these buttons in the view instead of in the controller?

**Comment by mennoo1996** [comment]
Except for two minor things, good work!

File: src/main/java/control/ToolViewController.java (hunk 2 above)
Comment (bart-de-jonge) [inline src/main/java/control/ToolViewController.java]: whoops

**Pull request merged**